src/status.js

```diff
@@ -1,8 +1,12 @@
-export default function checkBoxEventListener(parentCheckBoxContainer, tasksArray) {
+function checkBoxEventListener(parentCheckBoxContainer) {
+  const tasksArray = JSON.parse(localStorage.getItem('tasksList'));
   parentCheckBoxContainer.addEventListener('change', (event) => {
     const { target } = event;
     const index = tasksArray.findIndex((element) => element.index === parseInt(target.id, 10));
     tasksArray[index].completed = !tasksArray[index].completed;
-    localStorage.setItem('TasksList', JSON.stringify(tasksArray));
+    localStorage.setItem('tasksList', JSON.stringify(tasksArray));
   });
-}
\ No newline at end of file
+}
+
+// eslint-disable-next-line import/prefer-default-export
+export { checkBoxEventListener };
```

Comment:
- [ ] Let us fix eslint errors without disabling them, this makes your code looks unprofessional.